proto/optimint/optimint.proto

```diff
@@ -74,3 +74,13 @@ message Block {
 	Data data = 2;
 	Commit last_commit = 3;
 }
+
+message FraudProof {
+  uint64 block_height = 1;
+  repeated StateWitness state_witnesses = 2;
```

Comment:
I think it matches the research terminology better to have the `FraudProof` contain a single `StateWitness` that is itself an array of `key/value` pairs. Please correct me if I'm mistaken here.

Reply:
Added a `StateWitnessData` to consolidate the array of key/value pairs in a struct to reflect your comment

Reply:
I'd prefer to be able to retrieve a single `stateWitness` object that itself containers a slice of `witnessData`
See https://www.notion.so/celestiaorg/Fraud-Proof-Paper-Section-4-07714c3183f944ab8b98394fd3d61287#aaa8337929cb4fc5a8703bc67cb8a1e5

Something like:
```
message FraudProof {
  uint64 block_height = 1;
  StateWitness state_witness = 2;
}

message StateWitness {
   repeated WitnessData witness_data = 1;
}

message WitnessData {
  bytes key = 1;
  bytes value = 2;
}
```

Reply:
Made changes to reflect this suggestion